src/__tests__/Search/SelectedRevision.test.tsx

```diff
@@ -1,15 +1,20 @@
-import { renderHook } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { act } from 'react-dom/test-utils';
```

Comment:
note: it's incorrect to import `act` from `'react-dom/test-utils'` when using it with testing-library. We need to use the one exported by `testing-library` that does a little bit more stuff. It's reexported in `test-utils`.